test/lib/code_corps/comment/service_test.exs

```diff
@@ -92,6 +97,13 @@ defmodule CodeCorps.Comment.ServiceTest do
       refute_received({:patch, "https://api.github.com/" <> _rest, _body, _headers, _options})
     end
 
+    test "sets modified_from to 'code_corps'" do
+      comment = insert(:comment, modified_from: "github")
+      {:ok, updated_comment} = comment |> Comment.Service.update(@update_attrs)
+
+      assert updated_comment.modified_from == "code_corps"
+    end
```

Comment:
A changeset test might seem enough, but I wanted to add a sort of "integration" test to. The service is where the modification gets commited to the db.